spec/components/freedom_patches/seed_fu_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe "seed-fu patch" do
+  it "does not modify a sequence on an existing table" do
+    u = User.create!(username: "test1", email: "test1@example.com")
+    uid1 = u.id
+    UserDestroyer.new(Discourse.system_user).destroy(u)
+
+    SeedFu.quiet = true
+    SeedFu.seed
```

Comment:
Is this going to run all the seeds and make the test slow?

Reply:
Yes it will run all the seeds. But we already do this in a number of different specs. It's very fast:

```
    puts Benchmark.measure { SeedFu.seed }
```
```
  0.041236   0.002658   0.043894 (  0.052766)
```

Reply:
Hmm I’m surprised it is that fast. I guess seeds are already run during the initial migration of the test database so this call is like a noop?